src/monster_sbin.c

```diff
@@ -0,0 +1,1013 @@
+#include "global.h"
+#include "file_system.h"
+
+extern const File gMonsterFiles[];
+
+const FileArchive gMonsterFileArchive = {
+    .magic = "pksdir0",
+    .count = 497,
```

Comment:
Is there anyway we can get a NUMELEMS or something similar? I don't mind a constant/enum being used here. 

Just don't like the magic number here.

Reply:
Added